src/libraries/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs

```diff
@@ -337,6 +337,23 @@ public void Close(int timeout)
             Dispose();
         }
 
+        public void Close(TimeSpan timeout) => Close(ToTimeoutMilliseconds(timeout));
```

Comment:
per the docs, this is a timeout in SECONDS not milliseconds

Reply:
The good news is that once we have these new API's, customers won't be mixing up units any more (assuming we get them right in these PR's)